packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -217,16 +221,25 @@ export class DatabricksDriver extends JDBCDriver {
     }));
   }
 
+  private getFullDbName(dbName?: string) {
```

Comment:
Why do you need this as a separate method?

Reply:
used in two places, looks like a reasonable thing